File: moregorenine/baekjoon/step3_for/q4_15552/main.js
const fs = require('fs')
const stdin = (process.platform === 'linux'
        ? fs.readFileSync(0, "utf8")
        : `5
1 1
12 34
5 500
40 60
1000 1000`
).split('\n')

const count = parseInt(stdin[0])
showSum(count);

function showSum(count) {
    let twoNumbers
    let result = ""
    for (let i = 0; i < count; i++) {
        twoNumbers = stdin[i + 1].split(' ');
        result += (parseInt(twoNumbers[0]) + parseInt(twoNumbers[1])+'\n')
    }
    console.log(result)
}

File: moregorenine/baekjoon/step3_for/q4_15552/main.py
import sys


def setTwoNumbersUntilCount(count):
    twoNumbers = []

    for i in range(0, count):
        twoNumbers.append(sys.stdin.readline())

    return twoNumbers


def showSum(twoNumbers):
    for item in twoNumbers:
        twoNumber = item.split(" ")
        print(int(twoNumber[0]) + int(twoNumber[1]))


count = int(sys.stdin.readline())
twoNumbers = setTwoNumbersUntilCount(count)
showSum(twoNumbers)
